nixos/modules/services/databases/cockroachdb.nix

```diff
@@ -140,9 +139,11 @@ in
           data for SQL queries.
 
           This can be a percentage, expressed with a fraction sign or as a
-          decimal-point number, or any bytes-based unit. For example, "25%",
-          "0.25" both represent 25% of the available system memory. The values
-          "1000000000" and "1GB" both represent 1 gigabyte of memory.
+          decimal-point number, or any bytes-based unit. For example,
+          <literal>"25%"</literal>, <literal>"0.25"</literal> both represent
```

Comment:
Hm, maybe remove the double quotes as well, as `<literal/>` are already formatted in a different way in the manual.

Reply:
Do they actually put quotes, though? I should have double checked, I just left them to make it a bit more clear they're of `string` type.